### PR TITLE
chore: add confluence and firestore to definition yamls

### PR DESCRIPTION
**Pull request by harshithmullapudi**
## What
Adding confluence and firestore to source and destination definitions

### CODE REVIEW

**Comment by harshithmullapudi** [comment]
/publish-external connector=farosai/airbyte-pagerduty-source version=0.1.23

**Comment by harshithmullapudi** [comment]
/publish-external connector=farosai/airbyte-pagerduty-source version=0.1.23

**Comment by harshithmullapudi** [comment]
/public-external connector=farosai/airbyte-jenkins-source version=0.1.23

**Comment by harshithmullapudi** [comment]
/publish-external connector=farosai/airbyte-jenkins-source version=0.1.23

**Comment by harshithmullapudi** [comment]
/publish-external connector=ghcr.io/devmate-cloud/streamr-airbyte-connectors version=0.0.1

File: airbyte-config/init/src/main/resources/seed/destination_definitions.yaml [hunk 1]
```diff
@@ -63,6 +63,13 @@
   dockerImageTag: 0.1.20
   documentationUrl: https://docs.airbyte.io/integrations/destinations/gcs
   icon: googlecloudstorage.svg
+- name: Google Firestore
+  sourceDefinitionId: 27dc7500-6d1b-40b1-8b07-e2f2aea3c9f4
+  dockerRepository: airbyte/destination-firestore
+  dockerImageTag: 0.1.1
+  documentationUrl: https://docs.airbyte.io/integrations/destinations/firestore
```
Comment (alafanechere) [inline airbyte-config/init/src/main/resources/seed/destination_definitions.yaml:70]: I think we miss documentation for firestore, you could add a todo comment.

File: airbyte-config/init/src/main/resources/seed/source_specs.yaml [hunk 2]
```diff
@@ -3383,6 +3414,52 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
+- dockerImage: "farosai/airbyte-jenkins-source:0.1.23"
+  spec:
+    documentationUrl: "https://docs.faros.ai"
```
Comment (alafanechere) [inline airbyte-config/init/src/main/resources/seed/source_specs.yaml:3419]:
```suggestion
    documentationUrl: "https://github.com/faros-ai/airbyte-connectors"
```

File: docs/integrations/sources/confluence.md [hunk 3]
```diff
@@ -49,4 +49,5 @@ Follow [this article](https://support.atlassian.com/atlassian-account/docs/manag
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.1.1 | 2022-01-31 | [9831](https://github.com/airbytehq/airbyte/pull/9831) | chore: added spec to cache and added to source definition file |
```
Comment (alafanechere) [inline docs/integrations/sources/confluence.md:52]: Could you rephrase it to something less technical, easier to understand for connector users?

File: tools/bin/check_images_exist.sh [hunk 4]
```diff
@@ -28,7 +28,8 @@ checkNormalizationImages() {
 checkConnectorImages() {
   echo "Checking connector images exist..."
 
-  CONNECTOR_DEFINITIONS=$(grep "dockerRepository" -h -A1 airbyte-config/init/src/main/resources/seed/*.yaml | grep -v -- "^--$" | tr -d ' ')
+  # Added `: airbyte/` since we have added external images also and we don't' want to check those images
+  CONNECTOR_DEFINITIONS=$(grep "dockerRepository: airbyte/" -h -A1 airbyte-config/init/src/main/resources/seed/*.yaml | grep -v -- "^--$" | tr -d ' ')
```
Comment (alafanechere) [inline tools/bin/check_images_exist.sh:32]: Even if it's a small change, I think it could deserve a separate PR for traceability and awareness in the Airbyte changelog.

File: airbyte-integrations/connectors/source-confluence/Dockerfile [hunk 5]
```diff
@@ -34,5 +34,5 @@ COPY source_confluence ./source_confluence
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.version=0.1.1
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-confluence/Dockerfile:37]: Why are you bumping the version? I can't see any change in this PR on the confluence connector code.

**Review by alafanechere — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-confluence/Dockerfile (hunk 5 above)
Comment (harshithmullapudi) [reply airbyte-integrations/connectors/source-confluence/Dockerfile:37]: Yeah the spec is not in cache so I thought the ideal way is to bump it and let it go through the process rather than doing it manually

File: tools/bin/check_images_exist.sh (hunk 4 above)
Comment (harshithmullapudi) [reply tools/bin/check_images_exist.sh:32]: Yeah makes sense moved to a different PR

**Comment by harshithmullapudi** [comment]
/publish-external connector=faros-ai/airbyte-customer-io-source version=0.1.23

File: airbyte-config/init/src/main/resources/seed/destination_definitions.yaml [hunk 6]
```diff
@@ -63,6 +63,14 @@
...
   documentationUrl: https://docs.airbyte.io/integrations/destinations/gcs
   icon: googlecloudstorage.svg
+- name: Google Firestore
+  destinationDefinitionId: 27dc7500-6d1b-40b1-8b07-e2f2aea3c9f4
+  dockerRepository: airbyte/destination-firestore
+  dockerImageTag: 0.1.1
+  # todo - https://github.com/airbytehq/airbyte/issues/9727
+  documentationUrl: https://docs.airbyte.io/integrations/destinations/firestore
+  icon: firestore.svg
+  sourceType: api
```
Comment (alafanechere) [inline airbyte-config/init/src/main/resources/seed/destination_definitions.yaml:73]:
```suggestion
```
I don't think this is a valid field for a destination definition :)

File: airbyte-integrations/connectors/source-confluence/Dockerfile (hunk 5 above)
Comment (alafanechere) [reply airbyte-integrations/connectors/source-confluence/Dockerfile:37]: Sorry I don't get this: If you add the source connector with 0.1.0 to `source_definitions.yaml` and rerun ` ./gradlew airbyte-config:init:processResources` its not updating the `source_specs.yaml`? 
I think that version bumps should always be linked to a PR that modifies the code of the connector, which is not the case here: using 0.1.0 and 0.1.1 will be the same image.

**Review by alafanechere — CHANGES_REQUESTED** [review]

File: airbyte-config/init/src/main/resources/seed/destination_definitions.yaml (hunk 6 above)
Comment (harshithmullapudi) [reply airbyte-config/init/src/main/resources/seed/destination_definitions.yaml]: Yeah I figured out late 🙄

**Comment by harshithmullapudi** [comment]
/publish-external connector=farosai/airbyte-customer-io-source version=0.1.23

**Comment by harshithmullapudi** [comment]
/publish-external connector=farosai/airbyte-pagerduty-source version=0.1.23

**Comment by harshithmullapudi** [comment]
/publish-external connector=farosai/airbyte-harness-source version=0.1.23

**Comment by harshithmullapudi** [comment]
/publish-external connector=farosai/airbyte-victorops-source version=0.1.23
